src/js/server/attendance/logic.js

```diff
@@ -39,6 +51,54 @@ function sendTicketEmail(hacker) {
 }
 
 /**
+ * Get all invitations that are old enough to expire. Responses are hydrated with application and hacker objects
```

Comment:
Alas, I cannot take credit

http://stackoverflow.com/questions/6991135/what-does-it-mean-to-hydrate-an-object